nupic/research/connections.py

```diff
@@ -238,6 +238,54 @@ def numSynapses(self):
     return len(self._synapses)
 
 
+  def write(self, proto):
+    protoCells = proto.init('cells', self.numCells)
+
+    i = 0
+    for cell in xrange(self.numCells):
+      segments = self.segmentsForCell(cell)
+      protoSegments = protoCells[cell].init('segments', len(segments))
+
+      j = 0
```

Comment:
can you use enumerate in the for statement to avoid maintaining the index yourself? Same for `k` below